### ISSUE TITLE
Animated Stickers Not Send

### ISSUE THREAD

**Issue by ElG7dod** [An00nymushun/DiscordFreeStickers#29]
**Describe the bug**
When Try to Send Animated Sticker, It Appear Sent But not Actually sent to conversation and visible only to me as a real sticker.

**To Reproduce**
`Send Animated Sticker.`

1. Go to 'Stickers Menu'
2. Click on 'Animated Sticker'

**Expected behavior**
Sticker to Be Sent to Conversation As GIF.

**Screenshots**
![Screenshot](https://user-images.githubusercontent.com/60365002/212655424-fd54ec72-46a3-408a-8186-60556a8d9c9b.jpg)

**Client:**
 - Discord Build Channel [Release]
 - Plugin Version [1.4.0]
 - BD Version Core v1.8.4

**Additional context**
I Tried To Close all Plugins and keep only this plugin open and nothing work, and nothing appear in console when send sticker.

**Comment by FerLuisxd** [comment]
Same here

**Comment by FerLuisxd** [comment]
Same for me

**Comment by rastafaa** [comment]
Yeah doesn't work currently

**Comment by Alex-Paras** [comment]
same

**Comment by ArjhanToteck** [comment]
The gifs it renders seem to only loop once and not upload to discord correctly.

I tried opening them on photoshop and it says the file is invalid.

The render process seems to corrupt the gif somehow.

**Comment by VincentX0905** [comment]
same

**Comment by FerLuisxd** [comment]
I'm sad

**Comment by SkinChangerGD** [comment]
fix when

**Comment by ZorkIsEmo** [comment]
hey may of 2023 still not fixed

**Comment by LavenderAndChamomile** [comment]
They send for me then they disappear so the other person doesn't see them.

**Comment by Et3rnos** [comment]
I've been experimenting a bit with the plugin and I've found a way to send animated stickers, although they will be added as attachments to the current message, so you need to press Enter afterwards.

![image](https://github.com/An00nymushun/DiscordFreeStickers/assets/57318423/ccf0733e-351c-41d3-86e7-12b3be9a3fda)

Here is the fixed plugin:
https://gist.github.com/Et3rnos/b484cc59500638024031a2e677085271

Basically the `FileUploader.upload` function was broken (presumably because of a discord update) so I used `MessageAttachmentManager.addFiles` instead.

I'm not creating a pull request because I feel like there are better ways to support animated stickers without using attachments.

**Comment by SkinChangerGD** [comment]
> I've been experimenting a bit with the plugin and I've found a way to send animated stickers, although they will be added as attachments to the current message, so you need to press Enter afterwards.
> 
> ![image](https://user-images.githubusercontent.com/57318423/248204205-ccf0733e-351c-41d3-86e7-12b3be9a3fda.png)
> 
> Here is the fixed plugin: https://gist.github.com/Et3rnos/b484cc59500638024031a2e677085271
> 
> Basically the `FileUploader.upload` function was broken (presumably because of a discord update) so I used `MessageAttachmentManager.addFiles` instead.
> 
> I'm not creating a pull request because I feel like there are better ways to support animated stickers without using attachments.

thank you so much!

**Comment by ArjhanToteck** [comment]
> I've been experimenting a bit with the plugin and I've found a way to send animated stickers, although they will be added as attachments to the current message, so you need to press Enter afterwards.
> 
> ![image](https://github.com/An00nymushun/DiscordFreeStickers/assets/57318423/ccf0733e-351c-41d3-86e7-12b3be9a3fda)
> 
> Here is the fixed plugin:
> https://gist.github.com/Et3rnos/b484cc59500638024031a2e677085271
> 
> Basically the `FileUploader.upload` function was broken (presumably because of a discord update) so I used `MessageAttachmentManager.addFiles` instead.
> 
> I'm not creating a pull request because I feel like there are better ways to support animated stickers without using attachments.

you should make a pull request

**Comment by Et3rnos** [comment]
> > I've been experimenting a bit with the plugin and I've found a way to send animated stickers, although they will be added as attachments to the current message, so you need to press Enter afterwards.
> > ![image](https://user-images.githubusercontent.com/57318423/248204205-ccf0733e-351c-41d3-86e7-12b3be9a3fda.png)
> > Here is the fixed plugin:
> > https://gist.github.com/Et3rnos/b484cc59500638024031a2e677085271
> > Basically the `FileUploader.upload` function was broken (presumably because of a discord update) so I used `MessageAttachmentManager.addFiles` instead.
> > I'm not creating a pull request because I feel like there are better ways to support animated stickers without using attachments.
> 
> you should make a pull request

Okay then, here it is: https://github.com/An00nymushun/DiscordFreeStickers/pull/33
